airflow/config_templates/config.yml

```
@@ -2591,25 +2591,25 @@ sensors:
       type: float
       example: ~
       default: "604800"
-telemetry_collection:
+usage_data_collection:
   description: |
-    Airflow integrates `Scarf <https://about.scarf.sh/>`__ to collect basic telemetry data during operation.
-    This data assists Airflow maintainers in better understanding how Airflow is used.
+    Airflow integrates `Scarf <https://about.scarf.sh/>`__ to collect basic platform and usage data
+    during operation. This data assists Airflow maintainers in better understanding how Airflow is used.
     Insights gained from this telemetry are critical for prioritizing patches, minor releases, and
     security fixes. Additionally, this information supports key decisions related to the development road map.
     Check the FAQ doc for more information on what data is collected.
 
     Deployments can opt-out of analytics by setting the ``enabled`` option
-    to ``False``, or the ``SCARF_ANALYTICS=false`` environment variable.
+    to ``False``, or the ``USAGE_DATA_COLLECTION=false`` environment variable.
```

```suggestion
    to ``False``, or the ``SCARF_ANALYTICS=false`` environment variable.
```

This was intentional to support the env variable in https://docs.scarf.sh/package-analytics/ too

---

otherwise we didn't need other variable since we had Airflow config

---

sure ok 
i'll revert that one

---

@kaxil haha, had the same comment... seems to be a generic pitfall of reviewers. Shall a comment be added that you don't need to comment the next 10 times?

---

as though i would have read such a comment ;)

---

haha yeah worth adding a comment